src/shared/seccomp-util.c

```diff
@@ -89,6 +91,14 @@ int seccomp_add_secondary_archs(scmp_filter_ctx *c) {
 
 }
 
+bool is_seccomp_enabled(void) {
```

Comment:
hmm, i think "enabled" is a misnomer here, as this suggest that you could disable/enabled seccomp system-wide via some runtime setting, like you can do it for selinux.

I'd probably rename it to is_seccomp_available().